.github/workflows/ci.yml

```diff
@@ -168,7 +188,7 @@ jobs:
         if: ${{ matrix.unity-version != '2022.1.0a12' }}
         env:
           UNITY_VERSION: ${{ matrix.unity-version }}
-        run: dotnet msbuild /t:UnityPlayModeTest /p:Configuration=Release /p:OutDir=other
+        run: dotnet msbuild /t:UnityPlayModeTest /p:Configuration=Release /p:OutDir=other test/Sentry.Unity.Tests
```

Comment:
Added the project path explicitly so we're not running this target against the solution, and hence, against all projects in the solution. They are designed to spawn Unity once only